sycl/test-e2e/Plugin/interop-level-zero-buffer.cpp

```diff
@@ -2,8 +2,8 @@
 // L0 plugin incorrectly reports memory leaks because it doesn't take into
 // account direct calls to L0 API.
 // UNSUPPORTED: ze_debug-1,ze_debug4
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
-// RUN: env ONEAPI_DEVICE_SELECTOR='level_zero:*' ZE_DEBUG=1 %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %{build} %level_zero_options -o %t.out
```

Comment:
I believe this one will now need a `//REQUIRES: level_zero` directive on it, otherwise it could get run against OpenCL and other backends and devices.